### ISSUE TITLE
Debugging: Simulated routing speed can be too high, huge.

### ISSUE THREAD

**Issue by Rpnpif** [osmandapp/Osmand#5926]
With the Osmand development plugin, simulating the position of a calculated route gets a randomly routing speed that can be huge.
It could be until 500 km/h or more!
On some short route, the simulation could be almost unusable or not easy to use.

Expected: I suggest that the speed should vary not more than 10% to 120% of the maxspeed of the way with a increasing or decreasing curve more or less near the reality.

On some time, exceeding the maxspeed  is a good thing to test the speed sign icon apparition and the warning voice.

This is a minor issue but this feature is very handy for me, for example obviously to debug Osmand but also to debug the OSM map.

Regards.

**Comment by vshcherb** [comment]
Yes it is mostly for debug purposes and the speed parameter is needed in order to simulate it in real time
